includes/js/komoju-checkout-blocks.js

```diff
@@ -0,0 +1,102 @@
+const { useEffect, useCallback, useRef, createElement } = window.wp.element;
+
+function registerPaymentMethod(paymentMethod) {
+    let name = `${paymentMethod.id}`
+    const settings = window.wc.wcSettings.getSetting(`${name}_data`, {});
+
+    const komojuFields = createElement('komoju-fields', {
+        'token': '',
+        'name': 'komoju_payment_token',
+        'komoju-api': settings.komojuApi,
+        'publishable-key': settings.publishableKey,
+        'session': settings.session,
+        'payment-type': settings.paymentType,
+        style: { display: 'none' },
+    });
+
+    const label = createElement('div', {
+        style: { display: 'block', alignItems: 'center', justifyContent: 'center' }
+    },
+        window.wp.htmlEntities.decodeEntities(settings.title || window.wp.i18n.__('NULL GATEWAY', 'test_komoju_gateway')),
+        createElement('img', {
+            src: settings.icon,
+            alt: settings.title || 'Payment Method Icon',
+            style: { display: 'flex', alignItems: 'center', justifyContent: 'center', marginLeft: '10px' }
+        }),
+        komojuFields
+    );
+
+    const KomojuComponent = ({ activePaymentMethod, emitResponse, eventRegistration }) => {
+        const { onPaymentSetup } = eventRegistration;
+
+        useEffect(() => {
+            const komojuField = document.querySelector(`komoju-fields[payment-type='${paymentMethod.paymentType}']`);
```

Comment:
@Dinwy Is there a way we can set the locale for KOMOJU fields to the checkout language. For some reason it renders in Japanese for me on my English checkout page

![image](https://github.com/degica/komoju-woocommerce/assets/82835/eeb81515-3a11-4cac-99bd-5e80dfb911c1)

Reply:
Sure, there is locale option the komoju-field. Fixed in this commit.
[Commit](https://github.com/degica/komoju-woocommerce/pull/111/commits/62345a65041d0e7bb58fcd7c1a8ff0179309ccdc)

![image](https://github.com/degica/komoju-woocommerce/assets/16696001/388f1527-5b99-4c67-ad9e-453e825a8a67)